### DISCUSSION TITLE
Need to use aliasing on select

### DISCUSSION THREAD

**Discussion by thejaigoyal** [prisma/prisma#8505] [Q&A] [answered]
Hey Guys,

I'm using Prisma in a GraphQl service, I need to implement a query with aliasing on selected fields.

`select changedToStatus as status`

can anyone please tell me how to do it?

**Comment by ryands17** [comment, marked as answer]
@thejaigoyal 👋 

Currently this isn't possible directly in Prisma so you would need to rename this after you fetch the data. There's a request for the same so it would be great if you could add a 👍 to this.
https://github.com/prisma/prisma/issues/4433

**Reply by thejaigoyal** [reply]
Thanks for the quick response @ryands17, I hope the request for adding this functionality in Prisma will be addressed soon, mapping the field names after fetching them form DB, will affect the performance, so i think i'll go with the Raw Query for now instead.

**Reply by ryands17** [reply]
As you're using GraphQL, you don't need a raw query. You can simply map the field resolver as follows:

```ts
{
  status: (parent, args, context) => {
    return parent.changedToStatus;
  }
}
```

**Reply by thejaigoyal** [reply]
Oh yeah, thanks for the nice suggestion!! will implement it this way.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
